pacmod_interface/launch/pacmod_steer_test.launch.xml

```diff
@@ -4,6 +4,7 @@
 
   <arg name="vehicle_model" description="vehicle model name"/>
   <arg name="pacmod_param_path" default="$(find-pkg-share pacmod_interface)/config/pacmod_steer_test.param.yaml"/>
+  <arg name="socketcan_device" default="canVehicle"/>
```

Comment:
```suggestion
  <arg name="socketcan_device" default="can0"/>
```
Either one would be fine, but this is referred to https://github.com/tier4/jpntaxi_description/pull/35/files

Reply:
https://github.com/tier4/jpntaxi_description/pull/38 "canVehicle" is right.

Reply:
:pray: